fed/deliver.go

```diff
@@ -365,10 +376,9 @@ func (q *Queue) queueTasks(
 	if wideDelivery {
 		followers, err := q.DB.QueryContext(
 			ctx,
-			`select distinct follower from follows where followed = ? and follower not like ? and follower not like ? and accepted = 1`,
```

Comment:
`and not exists (select 1 from persons where persons.id = follows.follower and persons.id25519privkey is not null)`